tests/integration/test_backup_restore_on_cluster/configs/slow_replicated_merge_tree.xml

```diff
@@ -0,0 +1,10 @@
+<clickhouse>
+    <test>
+        <data_parts_exchange>
+            <delay_before_sending_part_ms>250</delay_before_sending_part_ms>
+        </data_parts_exchange>
+        <clone_replica>
+            <delay_before_removing_local_parts_ms>250</delay_before_removing_local_parts_ms>
```

Comment:
It's not easy to reproduce that bug, in my test it is reproduced quite often only because for that test I made a `ReplicatedMergeTree` table much slower.